src/database/schema.sql

```diff
@@ -229,6 +229,16 @@ CREATE TABLE Likes (
 	FOREIGN KEY (listing_id) REFERENCES Listings ON DELETE CASCADE
 );
 
+CREATE TABLE OrganisationLikes (
```

Comment:
Could you change this to lower caps singular organisationlike 

I will be refactoring everything to lower case singular after this PR

Reply:
Do remember to change the endpoint sql statements to singular as well